etcd3/client.py

```diff
@@ -1182,17 +1392,24 @@ def snapshot(self, file_obj):
         for response in snapshot_response:
             file_obj.write(response.blob)
 
+    # Remove utility functions from class namespace
```

Comment:
Let's not delete these - the _ is enough to stop people from depending on them

Reply:
My motivation for this change was that becoming a bound method doesn't work well with these function's signatures (i.e. `self` will be passed in as `payload`). Perhaps marking them as `staticmethod` down here would be a better solution to that problem?

Regardless, I'll remove this logic for now.